service/src/main/java/com/ge/predix/acs/monitoring/AcsDBHealthIndicator.java

```diff
@@ -45,6 +48,9 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(AcsDBHealthIndicator.class);
 
     @Autowired
+    private TitanMigrationManager migrationManager;
```

Comment:
@sanjeevchopra,
TitanMigrationManager bean is only initialized for "titan" spring profile - [@Profile("titan")](https://github.com/predix/acs/blob/develop/service/src/main/java/com/ge/predix/acs/privilege/management/dao/TitanMigrationManager.java#L17-L18).  Autowiring TitanMigrationManager here will fail AcsMonitoringControllerIT controller test for "non-titan" profile with NoSuchBeanDefinitionException